GameServerLib/GameObjects/Fountain.cs

```diff
@@ -40,7 +40,7 @@ public void Update(float diff)
             {
                 if (champion.Team != Team)
                 {
-                    continue;
+                    champion.Stats.CurrentHealth -= 5000;
```

Comment:
The behavior of the Fountain Turret is one which should be scripted, not designed into the system.